example/mm_jobman.sh

```diff
@@ -38,6 +40,8 @@ declare -a download_remote=()
 declare -a upload_local=()
 declare -a upload_remote=()
 opcenter=""
+entrypoint=""
+cwd="/home/ec2-user"
```

Comment:
Is it a good idea to set default as `$HOME` which will be whatever HOME folder it is? Possibly in your case $HOME will be `/home/ec2-user`

Reply:
Otherwise it looks fine. Seems we are stuck at some micromamba environment issue that I have not ran into on our end (desktop and HPC). The "this_file" script i shared with you earlier did the trick to get everything set for our HPC submission. I don't immediately know what the problem is with your non-interactive job. ...